### PR TITLE
Minor tweaks for Getting Started Guide [ci skip]

### PR DESCRIPTION
**Pull request by adityapandit17**
Some minor tweaks for Rails Getting started guide. 
Also added some rails 7 references.

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ After reading this guide, you will know:
 Guide Assumptions
 -----------------
 
-This guide is designed for beginners who want to get started with a Rails
+This guide is designed for beginners who want to get started with creating a Rails
```
Comment (lewispb) [inline guides/source/getting_started.md:21]: The sentence here reads just fine to me as it is.

File: guides/source/getting_started.md [hunk 2]
```diff
@@ -98,8 +98,9 @@ $ ruby --version
 ruby 2.7.0
 ```
 
-Rails requires Ruby version 2.7.0 or later. If the version number returned is
-less than that number (such as 2.3.7, or 1.8.7), you'll need to install a fresh copy of Ruby.
+Rails requires Ruby version 2.7.0 or later. It is preferred to use latest Ruby version.
```
Comment (lewispb) [inline guides/source/getting_started.md:101]: > It is preferred to use latest Ruby version.

Not always. Often times the very latest verion of Ruby has incompatibilities with particular gems.

File: guides/source/getting_started.md (hunk 1 above)
Comment (adityapandit17) [reply guides/source/getting_started.md:21]: Please read the full line or content.
We need to make it easier to read for beginner.

File: guides/source/getting_started.md (hunk 2 above)
Comment (adityapandit17) [reply guides/source/getting_started.md:101]: Yes I know but other rails docs suggested this.

**Review by zzak — APPROVED** [review]

**Pull request merged**